deployer/src/main/java/org/apache/brooklyn/entity/php/httpd/PhpHttpdSshDriver.java

```diff
@@ -236,18 +255,17 @@ private String configureHttpPort() {
     //TODO refactos using the strategy pattern
     private String installPhp() {
         log.debug("Installing PHP v", new Object[]{getEntity().getPhpVersion()});
-        if(getEntity().getPhpVersion().equals("5.4")){
+        if (getEntity().getPhpVersion().equals("5.4")) {
```

Comment:
By default 5.5 is installed. 5.4 requires a different installation process. Older version are not supported yet.